### DISCUSSION TITLE
problem with searching Arabic and Persian

### DISCUSSION THREAD

**Discussion by ali2429** [sumatrapdfreader/sumatrapdf#2660] [General]
This is a great app, thank you to its creators. But there is a problem in searching for words in Arabic and Persian, and you have to enter the word in the search box like English from left to right, for example, to find the word "رود" Type it "دور"
Is there a solution to this problem?

**Comment by GitHubRulesOK** [comment]
This issues is covered in https://github.com/sumatrapdfreader/sumatrapdf/issues/2655 and other links in that
